report_hotel_restaurant/LICENSE/LICENSE

```diff
@@ -0,0 +1,16 @@
+    OpenERP, Open Source Management Solution
```

Comment:
Cannot we use a v15 lewvel license file?
I recommend that to see v15 odoo core please.